nixos/modules/security/google_oslogin.nix

```
@@ -50,6 +50,9 @@ in
     systemd.packages = [ package ];
     systemd.timers.google-oslogin-cache.wantedBy = [ "timers.target" ];
 
+    # Without network this will fail to fetch users which causes sudo not to work.
+    systemd.services.google-oslogin-cache.after = [ "network.target" ];
```

```suggestion
    systemd.services.google-oslogin-cache = {
      after = [ "network-online.target" ];
      wants = [ "network-online.target" ];
    };
```

`man system.special`:
> Units that strictly require a configured network connection should pull in network-online.target (via a Wants= type dependency) and order themselves after it.